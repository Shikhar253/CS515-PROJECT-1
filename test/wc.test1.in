This is a basic test case.
It includes multiple lines.
