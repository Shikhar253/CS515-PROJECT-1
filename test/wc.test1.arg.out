 2 10 57 test/wc.test1.in
